Model/Sales/Repository/PackageRepository.php

```diff
@@ -93,48 +94,80 @@ public function fitInDigital_stamp()
             return false;
         }
 
-        if ($this->getWeight() > self::DEFAULT_WEIGHT) {
+        if ($this->isAllProductsFitInDigitalStamp() === false) {
+            return false;
+        }
+
+        if ($this->getWeight() > self::DEFAULT_DIGITAL_STAMP_WEIGHT) {
             return false;
         }
 
         return true;
     }
 
+    /**
+     * Set weight depend on product setting 'Fit in digital stamp' and weight from product
+     *
+     * @param \Magento\Quote\Model\Quote\Item[] $products
+     *
+     * @return $this
+     */
+    public function setFitInDigitalStampFromQuoteProducts($products)
+    {
+        if (empty($products)) {
+            return $this;
+        }
+
+        foreach ($products as $product) {
+            $fitIn = $this->getAttributesFitInOptions($product, 'digital_stamp');
+
+            if ($fitIn === null){
```

Comment:
if ($this->getAttributesFitInOptions($product, 'digital_stamp') === null){

Reply:
Done!